src/game_constants.h

```diff
@@ -97,6 +97,9 @@ constexpr int freezer = 23; // -5 Celsius
 constexpr int freezing = 32; // 0 Celsius
 }
 
+// Shelf life of corpse in hours. This should be kept same as raw flesh.
+constexpr int CORPSE_ROT_TIME = 24;
```

Comment:
[](url)
```suggestion
constexpr time_duration CORPSE_ROT_TIME = 24_hours;
```

Reply:
Trying to make it into time_duration just causes build to fail due to error. 
`error: 'time_duration' does not name a type; did you mean 'aim_location'?`

Reply:
So include `calendar.h`?

Reply:
You need also `#include "calendar.h"` where it's declared. Now that I think about it, `time_duration` should probably live in a header of its own (or in `units`), since the whole `calendar` might be too much of an include here, but that's definitely out of scope.

Reply:
#include "calendar.h" fixed that.
I'll leave moving time_duration around to someone else.

Reply:
> error: 'time_duration' does not name a type; did you mean 'aim_location'?

`time_duration`, living in a `time_duration`
where the people understand and dream of the perfect man
`aim_location` leads to sweet salvation
for the people, for the good, for mankind brotherhood